src/app/games/yinsh/YinshRules.ts

```diff
@@ -269,6 +275,31 @@ export class YinshRules extends Rules<YinshMove, YinshGameState, YinshLegalitySt
             });
         return captures;
     }
+    public getRingTargets(state: YinshGameState, start: Coord): Coord[] {
+        const targets: Coord[] = [];
+        for (const dir of HexaDirection.factory.all) {
+            let pieceSeen: boolean = false;
+            for (let cur: Coord = start.getNext(dir);
+                state.hexaBoard.isOnBoard(cur);
+                cur = cur.getNext(dir)) {
```

Comment:
let & state starting at same column
{ on f's column

Reply:
faudra revoir la règle du lint du coup, il m'affiche une erreur indentant state sur la même colonne que le let

Reply:
Ou bien, comme ça termine qu'à la ligne 111, tu garde les for en monoligne :p ?